lua/terrortown/lang/zh_hans.lua

```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
---L.label_keyhelper_free_roam = "leave object and roam free"
---L.label_keyhelper_flashlight = "toggle flashlight"
---L.label_keyhelper_quickchat = "open quickchat"
---L.label_keyhelper_voice_global = "global voice chat"
---L.label_keyhelper_voice_team = "team voice chat"
---L.label_keyhelper_chat_global = "global chat"
---L.label_keyhelper_chat_team = "team chat"
---L.label_keyhelper_show_all = "show all"
---L.label_keyhelper_disguiser = "toggle disguiser"
---L.label_keyhelper_save_exit = "save and exit"
---L.label_keyhelper_spec_third_person = "toggle third person view"
+L.help_keyhelp = [[
+按键绑定帮助器是用户界面元素的一部分，它总是向玩家显示相关的按键绑定，这对新玩家特别有帮助。按键绑定有三种不同类型：
+
+核心：它们包含 TTT2 中最重要的绑定。没有它们，游戏就很难发挥其全部潜力。
+额外：与核心绑定类似，但并非总是需要。它们包含聊天、语音或手电筒等内容。启用这些功能可能会对新玩家有所帮助。
+装备：有些装备项目有自己的绑定，这些都显示在此类别中。
+
+当记分板可见时，已禁用的类别仍会显示]]
+
+L.label_keyhelp_show_core = "启用始终显示核心绑定"
+L.label_keyhelp_show_extra = "启用始终显示额外绑定"
+L.label_keyhelp_show_equipment = "启用始终显示装备绑定"
+
+L.header_interface_keys = "按键绑定帮助器设置"
+L.header_interface_wepswitch = "武器切换UI设置"
+
+L.label_keyhelper_help = "打开游戏模式菜单"
+L.label_keyhelper_mutespec = "切换观看语音模式"
+L.label_keyhelper_shop = "打开装备商店"
+L.label_keyhelper_show_pointer = "显示鼠标指针"
+L.label_keyhelper_possess_focus_entity = "附体焦点中的Prop"
+L.label_keyhelper_spec_focus_player = "观看焦点中的玩家"
+L.label_keyhelper_spec_previous_player = "上一个玩家"
+L.label_keyhelper_spec_next_player = "下一个玩家"
+L.label_keyhelper_spec_player = "观看玩家"
+L.label_keyhelper_possession_jump = "Prop：跳跃"
+L.label_keyhelper_possession_left = "Prop：向左"
+L.label_keyhelper_possession_right = "Prop：向右"
+L.label_keyhelper_possession_forward = "Prop：向前"
+L.label_keyhelper_possession_backward = "Prop：向后"
+L.label_keyhelper_free_roam = "离开对象并自由漫游"
+L.label_keyhelper_flashlight = "切换手电筒"
+L.label_keyhelper_quickchat = "打开快速聊天"
+L.label_keyhelper_voice_global = "全局语音聊天"
+L.label_keyhelper_voice_team = "团队语音聊天"
+L.label_keyhelper_chat_global = "全局聊天"
+L.label_keyhelper_chat_team = "团队聊天"
+L.label_keyhelper_show_all = "显示全部"
+L.label_keyhelper_disguiser = "切换伪装"
+L.label_keyhelper_save_exit = "保存退出"
+L.label_keyhelper_spec_third_person = "切换第三人称视图"
 
 -- 2023-10-26
---L.item_armor_reinforced = "Reinforced Armor"
---L.item_armor_sidebar = "Armor protects you against bullets penetarating your body. But not forever."
---L.item_disguiser_sidebar = "Armor protects you against bullets penetarating your body. But not forever."
---L.status_speed_name = "Speed Multiplier"
---L.status_speed_description_good = "You are faster than normal. Items, equipment or effects can influence this."
---L.status_speed_description_bad = "You are slower than normal. Items, equipment or effects can influence this."
-
---L.status_on = "on"
---L.status_off = "off"
-
---L.crowbar_help_primary = "Attack"
---L.crowbar_help_secondary = "Push players"
+L.item_armor_reinforced = "强化装甲"
+L.item_armor_sidebar = "盔甲保护你免受子弹穿透身体。但不是永远。"
+L.item_disguiser_sidebar = "盔甲可防止子弹穿透身体。但不是永远。"
+L.status_speed_name = "速度倍增器"
```

Comment:
```suggestion
L.status_speed_name = "移速倍率"
```

"Multiplier" is referring to the mathematical operation and not a device.